draft-ietf-masque-connect-udp.md

```diff
@@ -45,119 +55,222 @@ document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
 In this document, we use the term "proxy" to refer to the HTTP server that
-opens the UDP socket and responds to the CONNECT-UDP request. If there are
-HTTP intermediaries (as defined in Section 2.3 of {{RFC7230}}) between the
+opens the UDP socket and responds to the UDP proxying request. If there are
+HTTP intermediaries (as defined in {{Section 3.7 of SEMANTICS}}) between the
 client and the proxy, those are referred to as "intermediaries" in this
 document.
 
+Note that, when the HTTP version in use does not support multiplexing streams
+(such as HTTP/1.1), then any reference to "stream" in this document is meant to
+represent the entire connection.
 
-# Supported HTTP Versions {#http-versions}
 
-The CONNECT-UDP method is defined for all versions of HTTP. UDP payloads are
-sent using HTTP Datagrams {{!HTTP-DGRAM=I-D.ietf-masque-h3-datagram}}. Note
-that, when the HTTP version in use does not support multiplexing streams (such
-as HTTP/1.1), then any reference to "stream" in this document is meant to
-represent the entire connection.
+# Configuration of Clients {#client-config}
+
+Clients are configured to use UDP Proxying over HTTP via an URI Template
+{{!TEMPLATE=RFC6570}}. The URI template MUST contain exactly two variables:
+"target_host" and "target_port". Examples are shown below:
+
+~~~
+https://masque.example.org/{target_host}/{target_port}/
+https://proxy.example.org:4443/masque?h={target_host}&p={target_port}
```

Comment:
Hm, I think if we're defining variables in the template, it should look more like what DoH uses, and what the URI template RFC lays out. 

```suggestion
https://proxy.example.org:4443/masque{?target_host,target_port}
```

Reply:
Not necessarily, that would also be valid but so is this example. The example you describe would force the query parameter to be equal to the variable name, whereas the current example shows that you can pick custom query parameters

Reply:
Okay. I have a preference for the form I suggested, so there would be a consistent variable name in the URI.

Reply:
Fair enough, I've added that as another example line so people see it as an option